apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -511,6 +511,7 @@ export const realtime = {
   items: [
     { name: 'Overview', url: '/guides/realtime', items: [] },
     { name: 'Quickstart', url: '/guides/realtime/quickstart', items: [] },
+    { name: 'Protocol Description', url: '/guides/realtime/protocol', items: [] },
```

Comment:
Let's just change from `Protocol Description` to `Protocol` yeah?

Reply:
I like the more with description, but I don't want to argue because this PR takes a lot of time